src/System.IO.FileSystem.Primitives/tests/System.IO.FileSystem.Primitives.Tests.builds

```diff
@@ -4,7 +4,11 @@
   <ItemGroup>
     <Project Include="System.IO.FileSystem.Primitives.Tests.csproj"/>
     <Project Include="System.IO.FileSystem.Primitives.Tests.csproj">
-      <TestTFMs>netcore50;net46</TestTFMs>
+      <TargetGroup>netstandard1.7</TargetGroup>
+      <TestTFMs>netcoreapp1.1</TestTFMs>
```

Comment:
Looks like @joperezr PR got in before yours. Can you please adjust this .builds file and csproj such that the netstandard1.7 is the default configuration. See https://github.com/dotnet/corefx/pull/12241 as an example